awx/main/models/credential/__init__.py

```diff
@@ -975,6 +976,38 @@ def create(self):
     }
 )
 
+ManagedCredentialType(
+    namespace='github_token',
+    kind='token',
+    name=ugettext_noop('Github Personal Access Token'),
+    managed_by_tower=True,
+    inputs={
+        'fields': [{
+            'id': 'token',
+            'label': ugettext_noop('Token'),
```

Comment:
it's also possible to [add help_text to fields](https://github.com/ansible/awx/blob/devel/docs/credentials/custom_credential_types.md) here, and that might also help. Presumably this comes from somewhere in the github repo settings??